src/app.js

```diff
@@ -1,34 +1,55 @@
 import './style.css';
+import LiElement from './components/listElements';
+import { getScores, addScores } from './components/scores';
 import displayForm from './components/form';
-import { getScore, addScore } from './components/scores';
-import liElement from './components/listElements';
-import form from './components/form';
-
 
 const displayData = () => {
-  const data = getScore();
-  const sl = document.querySelector('.score-result');
-  sl.innerHTML = '';
-  data.forEach((score) => {
-    const li = liElement(score);
-    sl.append(li);
-  });
+  const sl = document.getElementById('score-result');
+  const response = getScores();
+  let data = [];
+  const liLoading = LiElement({ user: 'Getting data from API', score: '' });
+  sl.append(liLoading);
+
+  response
+    .then((response) => {
+      data = [...response.data.result];
+      sl.innerHTML = '';
+      data.forEach((score) => {
+        const li = LiElement(score);
+        sl.append(li);
+      });
+    })
+    .catch((error) => {
+      const liLoading = LiElement({
+        user: "Couldn't get data from API :(",
+        score: '',
+      });
+      sl.append(liLoading);
+    });
 };
 
 const submitScore = (e) => {
   e.preventDefault();
-  const name = document.getElementById('name').value;
+  const user = document.getElementById('name').value;
   const score = document.getElementById('score').value;
-  const obj = { name, score};
-  
-  addScore(obj);
+  const game = { user, score };
+
+  const response = addScores(game);
+  response.then((resp) => {
+    console.log(resp);
+  });
   displayData();
-  form.reset();
-}
+};
```

Comment:
- [ ] The submit button doesn't work or hit the POST API endpoint and post user entries, on my end, the button is not clickable. please fix it.
![data-14](https://user-images.githubusercontent.com/55635977/141657849-0522d0a3-556e-4c35-80db-59ad23bc7148.png)

The submit button should hit the POST API endpoint and post user entries.

- [ ] You can submit user data and it reflects on the scoreboard. Please make sure that the user sends the data so that it should be reflected on the scoreboard.